library/src/main/java/com/novoda/downloadmanager/DownloadBatchNotification.java

```diff
@@ -26,7 +25,9 @@ public NotificationInformation createNotification(DownloadBatchTitle downloadBat
         String title = downloadBatchTitle.asString();
         String content = percentageDownloaded + "% downloaded";
 
-        Notification notification = new NotificationCompat.Builder(context, CHANNEL_ID)
+        String notificationChannel = NotificationChannelCreator.createDownloadNotificationChannel(context);
```

Comment:
I think that `NotificationChannelCreator` should be private. The reason is that this `DownloadBatchNotification` is an implementation of the public `NotificationCreator`.

A client can use just the default notification (and so this default channel), or if they want to create their own notification they should handle it all and decide if they want a channel or not, and with which features (showBadge? lights? etc...)

On the other hand, if you make it public then it belongs to the API contract and we'll have to maintain it and be more strict and careful with the changes, even changes on the class name.